.github/workflows/test.yml

```diff
@@ -6,6 +6,7 @@ on:
     branches: [ master ]
   schedule:
     - cron: '0 */12 * * *'
+  workflow_dispatch:
```

Comment:
how can we manually trigger it from UI? is it button click?

Reply:
yeah, this adds a "run workflow" button to click.